pyglmnet/pyglmnet.py

```diff
@@ -5,6 +5,7 @@
 
 import numpy as np
 from scipy.special import expit
+from sklearn.base import BaseEstimator, RegressorMixin
```

Comment:
oh my bad... i didn't realize that sklearn dependency need to be out... for cross validation you only need a `fit` method and a `score` method if `scores=None`. I'll fit